### ISSUE TITLE
phpMyAdmin problem with Ubuntu 15.10 with PHP7

### ISSUE THREAD

**Issue by TOxicFre4k** [phpmyadmin/phpmyadmin#12131]
### Steps to reproduce
1. Install Php7 and MYSQL with Apache2
2. Try using sudo apt-get install phpmyadmin
3. It still shows only php5 dependencies, although phpmyadmin 4.6.0 is available with php7 support(as given in site)

### Expected behaviour
phpmyadmin installs successfully and later configuration options should be shown!

### Actual behaviour
Shows dependencies are broken and cannot be fixed. Shows support is only for PHP5

### Server configuration
Ubuntu 15.10

**Web server:**
Apache Web Server 2
**Database:**
MYSQL
**PHP version:**
7.0
**phpMyAdmin version:**
No idea which php version is defaut for terminal package release of php
### Client configuration
**Browser:**

**Operating system:**

**Comment by ibennetch** [comment]
Hi,

When using the distribution-provided package, you're telling the package manager to handle the installation of the package you've selected and all dependencies. This is good for users who don't want to, for instance, install their own PHP and want to let dpkg/apt manage it for them, and provides an easy way for the package manager to make sure all dependencies are at their proper version and correctly installed. However, the downside for situations like yours is that it means you're stuck using the package-provided version of all dependencies, you can't easily mix and match. There are ways to fake apt in to not installing dependencies (such as http://askubuntu.com/questions/74523/how-can-i-install-a-package-without-installing-some-dependencies) but that's way beyond the scope of what we are able to support.

Basically, you can either install your own phpMyAdmin to go with your own PHP version, or you can remove your own PHP so that the package manager can install their version of PHP. Just because the official phpMyAdmin source version works with PHP7 doesn't mean distributions have to immediately begin packaging PHP7, and in Ubuntu's case they only provide phpMyAdmin 4.4.13.1 anyway.

Long story short, you have to decide whether you want to allow dpkg/apt to manage your package and dependencies or if you want to do it yourself, then apply that to the entire stack (or do the advanced technique to override that behavior).
